app/assets/stylesheets/layout.css

```diff
@@ -1,6 +1,15 @@
 .l-relative {
   position: relative; }
 
+.l-flex {
+  display: flex; }
+
+.l-mb-4 {
+  margin-bottom: 1rem; }
+
+.l-mr-2 {
+  margin-right: 0.5rem; }
+
```

Comment:
this is heavily inspired by tailwindcss, I think it makes creating a layout much easier